packages/nc-gui/components/virtual-cell/components/ListItems.vue

```diff
@@ -98,6 +103,29 @@ const newRowState = computed(() => {
 watch(expandedFormDlg, (nexVal) => {
   if (!nexVal && !isNew.value) vModel.value = false
 })
+
+useSelectedCellKeyupListener(vModel, (e: KeyboardEvent) => {
+  switch (e.key) {
+    case 'ArrowUp':
+      selectedRowIndex.value = Math.max(0, selectedRowIndex.value - 1)
+      e.stopPropagation()
+      break
+    case 'ArrowDown':
+      selectedRowIndex.value = Math.min(childrenExcludedList.value?.list?.length - 1, selectedRowIndex.value + 1)
+      e.stopPropagation()
+      break
+    case 'Enter':
+      const selectedRow = childrenExcludedList.value?.list?.[selectedRowIndex.value]
+      if (selectedRow) {
+        linkRow(selectedRow)
+        e.stopPropagation()
+      }
+      break
+  }
+})
+const activeRow = (el: InstanceType<typeof Card>) => {
+  el?.$el?.scrollIntoView({ block: 'nearest', inline: 'nearest' })
```

Comment:
It should not be `el?` - as mentioned before, using `?` everywhere makes the code *less* clean as it suggests "I have typed el as `InstanceType<typeof Card>` but actually I am not sure if it's defined or not"

If you're unsure if `el` is going to be defined, then type it as such, i.e. `(el?: InstanceType<typeof Card>)`

Otherwise it's just making it a lot harder to understand what is actually defined and what *could* be defined.

Reply:
It won't be defined always, I will update the type :+1: